example/.aspect/cli/config.yaml

```diff
@@ -1,5 +1,9 @@
 plugins:
   - name: lint-plugin
     # For this to work, you must first `bazel build dev` in the root of the repo
-    from: ../../bazel-bin/plugin
+    from: ../bazel-bin/plugin
     log_level: warn
+    properties:
+      aspects:
```

Comment:
This is being parsed into a `LintAspects` property, why not name it the same here?

Reply:
I was on the fence which name to use. WDYT? `lint_aspects` here, or `Aspects` in the Go code?

Reply:
For the quickest solution to avoid thinking about that... I'd say `lint_aspects` describes exactly what it is so I'd prefer that out of those 2 options.

But will the person writing this file always be the one writing the aspects? Do we even need the word `aspect`? I guess we're also exposing the `%` syntax here so idk if it's worth thinking about too much atm.

Reply:
I guess we are just blindly inserting these as `--aspects` so in theory someone could pass any aspect here and it might have nothing to do with linting, although we might not want to "allow" that.

It could be something such as:
```yasml
linters:
    eslint@//:lint.bzl
```
or some magic like that to hide all "aspect" logic from this file.

Reply:
I don't think we're going to hide it that well. You also have to write a `lint.bzl` file to declare the aspects we use. So I think it's okay `lint_aspects`

Reply:
I just wonder if it would be different people writing aspects vs deciding which to use? `lint_aspects` is clearest atm though 👍